monkey/tests/unit_tests/monkey_island/cc/services/telemetry/processing/credentials/conftest.py

```diff
@@ -3,17 +3,27 @@
 import mongoengine
 import pytest
 
+from monkey_island.cc.models import Monkey
 from monkey_island.cc.services.config import ConfigService
 
+fake_monkey_guid = "272405690278083"
+fake_ip_address = "192.168.56.1"
+
 
 @pytest.fixture
-def fake_mongo(monkeypatch):
+def fake_mongo(monkeypatch, uses_encryptor):
```

Comment:
The `uses_encryptor` fixture is super slow. We need to add `@pytest.mark.slow` to any test that uses the `fake_mongo` fixture.